[INFO] 2019-02-16 17:06:43,584 >>	Classifier Type: PYTORCH
[INFO] 2019-02-16 17:06:43,585 >>	Input Path: /home/berry/Malware/data/subset/min
[INFO] 2019-02-16 17:06:43,586 >>	--------------------------------
[INFO] 2019-02-16 17:06:43,587 >>	[START]  Processor.preprocess()
[INFO] 2019-02-16 17:06:43,588 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,606 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,607 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,609 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,611 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,614 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,615 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,618 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,619 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,622 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,623 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,626 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,627 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,630 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,631 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,633 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,635 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,638 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,639 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,642 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,643 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,645 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,647 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,650 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,651 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,653 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,655 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,657 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,659 >>	[START]  FeatureExtractor.extract()
[INFO] 2019-02-16 17:06:43,662 >>	[FINISH] FeatureExtractor.extract(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,662 >>	[FINISH] Processor.preprocess(): [elapsed_time] >> 00:00:00
[INFO] 2019-02-16 17:06:43,664 >>	[START]  Processor.train()
[INFO] 2019-02-16 17:06:43,665 >>	[START]  PyTorchClassifier.train()
[INFO] 2019-02-16 17:06:43,668 >>		 >> 0 folds start
[INFO] 2019-02-16 17:06:43,669 >>		 	 Generating datasets...
[INFO] 2019-02-16 17:06:43,671 >>		 	 	 Reading train_Kernel02.csv...
[INFO] 2019-02-16 17:06:43,699 >>		 	 	 Reading train_Binary.csv...
[INFO] 2019-02-16 17:07:06,425 >>		 	 	 Reading validate_Kernel02.csv...
[INFO] 2019-02-16 17:07:06,442 >>		 	 	 Reading validate_Binary.csv...
[INFO] 2019-02-16 17:07:30,160 >>		 	 Datasets were generated.
[ERROR] 2019-02-16 17:14:49,229 >>	Unexpected Exception Occurred.
Traceback (most recent call last):
  File "src/main.py", line 23, in <module>
    main(get_option())
  File "src/main.py", line 14, in main
    ProcessorFactory.process(args)
  File "/home/berry/Malware/src/processor_factory.py", line 39, in process
    validity, predict = processor.process()
  File "/home/berry/Malware/src/processor.py", line 42, in process
    validity = None if only_prediction() and model_exist() else self.__train(feature_names)
  File "/home/berry/Malware/src/save_log.py", line 112, in wrapper
    result = func(*args, **kargs)
  File "/home/berry/Malware/src/processor.py", line 69, in __train
    return self.classifier.train(feature_names)
  File "/home/berry/Malware/src/save_log.py", line 112, in wrapper
    result = func(*args, **kargs)
  File "/home/berry/Malware/src/classifier/pytorch_classifier.py", line 94, in train
    verbose=self.params["verbose"])
  File "/home/berry/Malware/src/classifier/torch_util.py", line 75, in train_model
    loss = criterion(outputs, labels)
  File "/home/berry/.pyenv/versions/3.6.6/lib/python3.6/site-packages/torch/nn/modules/module.py", line 489, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/berry/.pyenv/versions/3.6.6/lib/python3.6/site-packages/torch/nn/modules/loss.py", line 504, in forward
    return F.binary_cross_entropy(input, target, weight=self.weight, reduction=self.reduction)
  File "/home/berry/.pyenv/versions/3.6.6/lib/python3.6/site-packages/torch/nn/functional.py", line 2027, in binary_cross_entropy
    input, target, weight, reduction_enum)
RuntimeError: reduce failed to synchronize: device-side assert triggered
